_pages/ato/types.md

```diff
@@ -3,17 +3,13 @@ title: Types of ATO
 navtitle: Types
 ---
 
-Choosing an authorization is an important decision. Work with your Infrastructure Lead and Product Owner to determine which authorization you will employ, and when. The Infrastructure Lead makes all final decisions and answers in writing (email, Slack, etc) any questions the team may have on how to meet the below objectives. Many situations may have no "right answer". It is up to the Infrastructure Lead to make a risk-based decision on what to do.
+In most cases, the types of ATO that will be pursued for GSA systems are the *GSA Lightweight ATO (LATO)*. Systems that are under development can be operated without an ATO provided they fulfill the requirements for [pre-assessment for internal government use](#conditions-for-pre-assessment).
 
-In most cases, the two types of ATO that will be pursued for GSA systems are the *GSA Lightweight ATO (LATO)* and the *FedRAMP Agency ATO*. Systems that are under development can be operated without an ATO provided they fulfil the requirements for [pre-assessment for internal government use](#conditions-for-pre-assessment).
+The GSA LATO is designed for systems built using agile methods that run on top of cloud infrastructure which has already received an ATO (such as AWS, Azure, and [cloud.gov](https://cloud.gov)). It's "lightweight" because only 24 NIST controls must be implemented in order to grant this type of ATO. The GSA LATO is a good option for systems 18F intends to own and operate operate.
 
-The [GSA LATO](#gsa-lato) is designed for systems built using agile methods that run on top of cloud infrastructure which has already received an ATO (such as AWS, Azure, and soon [cloud.gov](https://cloud.gov)). It's "lightweight" because only 24 NIST controls must be implemented in order to grant this type of ATO. It is initially valid for 90 days, but can be extended for longer periods with additional work. The GSA LATO is a good option for systems we intend to own and operate operate ourselves.
+The GSA LATO process is described in a procedural guide which can be accessed from [Insite](https://insite.gsa.gov/portal/content/627230) (search for "14-68" on that page). The GSA LATO is initially valid for 90 days. With additional work it can be extended up to a 1 year period for a Moderate impact system, or up to a 3 years period for a Low impact system. Please read the LATO guide before starting the LATO process.
```

Comment:
Also, is the 90-day a required step to get to the longer ones? As I've mentioned before, I think we should avoid doing 90-day LATOs to avoid duplicate effort.